packages/backend/src/tests/e2e/payments.test.ts

```diff
@@ -31,7 +32,7 @@ describe('e2e', () => {
             SELECT
                 payment_id,
                 amount,
-                payment_date
+                to_json(payment_date::timestamptz) as payment_date -- // TODO: fix this line
```

Comment:
Please add more context. This comment is not very helpful if we come back to fix this bug in 2 weeks. We will have forgotten everything and the  lack of context just makes this useless.

Reply:
It's like getting a bug report saying: "something isn't working".